src/utils/errorHandler.test.tsx

```diff
@@ -11,23 +11,88 @@ jest.mock('react-toastify', () => ({
 
 describe('Test if errorHandler is working properly', () => {
   const t: TFunction = (key: string) => key;
-  const tErrors: TFunction = (key: string, options?: Record<string, unknown>) =>
-    key;
+  const tErrors: TFunction = (
+    key: string,
+    options?: Record<string, unknown>,
+  ) => {
+    if (options) {
+      console.log(`options are passed, but the function returns only ${key}`);
+    }
+    return key;
+  };
```

Comment:
If i remove the console.log statement, it will give error for keeping the "options" argument unused. It is better to use is as log when it could be sent with parameters, but not used in whole codebase.